src/test/java/core/basesyntax/ActivityStrategyImplTest.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.strategy.ActivityHandler;
+import core.basesyntax.service.strategy.ActivityStrategyImpl;
+import core.basesyntax.service.strategy.BalanceActivityHandler;
+import core.basesyntax.service.strategy.PurchaseActivityHandler;
+import core.basesyntax.service.strategy.ReturnActivityHandler;
+import core.basesyntax.service.strategy.SupplyActivityHandler;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ActivityStrategyImplTest {
+    private ActivityStrategyImpl activityStrategy;
+
+    @BeforeEach
+    public void setUp() {
+        activityStrategy = new ActivityStrategyImpl(getActivitiesServiceMap());
+    }
+
+    @Test
+    public void strategy_testNull() {
+        try {
+            activityStrategy.getQuantityModifier(null);
+            fail("Expected IllegalArgumentException, but got no exception");
+        } catch (IllegalArgumentException e) {
+            assertEquals("Activity type cannot be null", e.getMessage());
+        }
+    }
+
+    @Test
+    public void testGetQuantityModifier_Success() {

```

Comment:
incorrect naming for test (fix it for all your test)
see [common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/solid/fruit-shop-tests#remember-about-naming-of-test-methods)